proto/prysm/v1alpha1/attestation/aggregation/attestations/maxcover.go

```diff
@@ -267,15 +267,28 @@ func (al attList) filterContained() (attList, error) {
 	})
 	filtered := al[:0]
 	filtered = append(filtered, al[0])
+
+	candidates := make([]*bitfield.Bitlist64, len(al))
+	for i := 0; i < len(al); i++ {
+		var err error
+		candidates[i], err = al[i].AggregationBits.ToBitlist64()
+		if err != nil {
+			return nil, err
+		}
+	}
+	coveredBits := bitfield.NewBitlist64(candidates[0].Len())
+	coveredBits.NoAllocOr(candidates[0], coveredBits)
+
 	for i := 1; i < len(al); i++ {
-		c, err := filtered[len(filtered)-1].AggregationBits.Contains(al[i].AggregationBits)
+		c, err := coveredBits.Contains(candidates[i])
 		if err != nil {
 			return nil, err
 		}
 		if c {
 			continue
 		}
 		filtered = append(filtered, al[i])
+		coveredBits.NoAllocOr(candidates[i], coveredBits)
```

Comment:
You will need to handle errors for `coveredBits.NoAllocOr` or the linter won't let this merge.

Reply:
Thank you for letting me know :)